one-pager/orchester/generate_pdf.py

```
@@ -0,0 +1,94 @@
+from io import BytesIO
+from google.cloud import storage
+from datetime import timedelta
+import google.auth
+import google.auth.transport.requests as g_requests
+
+
+# Configuración de GCP
+PROJECT_ID = "valiant-complex-467519-d3"
+GCS_BUCKET_NAME = "ithaka-one-pagers"
+
+def generate_pdf(texto_en_markdown, url_imagen, output_filename="output.pdf"):
```

Ruff (0.12.2)

Missing return type annotation for public function `generate_pdf`

(ANN201)